cms/templates/export.html

```diff
@@ -62,18 +64,18 @@ <h1 class="page-header">
           <h2 class="title">${_("About Exporting Libraries")}</h2>
           <div class="copy">
               ## Translators: ".tar.gz" is a file extension, and should not be translated
-              <p>${_("You can export libraries and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the library structure and content. You can also re-import libraries that you've exported.").format(
+              <p>${Text(_("You can export libraries and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the library structure and content. You can also re-import libraries that you've exported.").format(
           studio_name=settings.STUDIO_SHORT_NAME,
-          )}</p>
+          ))}</p>
           </div>
         %else:
           <h2 class="title">${_("About Exporting Courses")}</h2>
           <div class="copy">
               ## Translators: ".tar.gz" is a file extension, and should not be translated
-              <p>${_("You can export courses and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the course structure and content. You can also re-import courses that you've exported.").format(
+              <p>${Text(_("You can export courses and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the course structure and content. You can also re-import courses that you've exported.").format(
                 studio_name=settings.STUDIO_SHORT_NAME
-              )}</p>
-              <p>${_("{em_start}Caution:{em_end} When you export a course, information such as MATLAB API keys, LTI passports, annotation secret token strings, and annotation storage URLs are included in the exported data. If you share your exported files, you may also be sharing sensitive or license-specific information.").format(em_start='<strong>', em_end="</strong>")}</p>
+              ))}</p>
+              <p>${Text(_("{em_start}Caution:{em_end} When you export a course, information such as MATLAB API keys, LTI passports, annotation secret token strings, and annotation storage URLs are included in the exported data. If you share your exported files, you may also be sharing sensitive or license-specific information.")).format(em_start=HTML('<strong>'), em_end=HTML("</strong>"))}</p>
```

Comment:
Here you do need `Text()` as you have it.  I think it is simpler to code review if you have any arguments sent to `format()` on their own lines.